Gemfile

```diff
@@ -13,6 +13,7 @@ gem 'friendly_id', '~> 5.2', '>= 5.2.3'
 gem 'geokit-rails', '~> 2.3'
 gem 'health_check', '~> 2.7'
 gem 'high_voltage', '~> 3.0'
+gem 'jbuilder', '~> 2.7'
```

Comment:
[Jbuilder](https://github.com/rails/jbuilder) is like [Builder](https://github.com/jimweirich/builder) but for JSON.